hosts/voyager/iss.py

```diff
@@ -391,3 +391,42 @@ def initialize(asof=None):
 ISS.initialize()
 
 ################################################################################
+
+
+
+################################################################################
```

Comment:
I think horizontal separators are useful, so I'd like to come to a consensus on how to use them.  Generally, within the bulk of a file, Mark uses #### and I use ==== and **** for functions and classes.  I don't care what we use, but I'd like to standardize this so I can stop guessing / imposing my own conventions.  Rob has suggested not delineating methods, but what about classes?

Reply:
I will let Joe and Mark decide on what they want for this code base. I notice that Mark does put a horizontal divider before every class method in polymath and oops. I personally don't use any (I think between color coding and obvious doc strings the start of a method is easy to see) but I don't need to force that on others if the consensus is they are useful.

Reply:
I'm confused, because I am not seeing any changes since last time. There should never be three blank lines in a row anywhere. There should only be two (at most) before classes and some top-level function defs. There should never be two horizontal lines with only blank lines between them.